### ISSUE TITLE
Cannot expand the OPC node in some OPC UA server

### ISSUE THREAD

**Issue by ryan-ms** [Azure/Industrial-IoT#110]
**Describe the bug**

Cannot successfully expand the OPC node in some OPC UA server while other OPC UA client can successful expand it.

When the problem occurred, the response from OPC Twin service would show error message: "`BadNodeIdUnknown`". 

**To Reproduce**
1.  Runniung OPC Twin and OPC Publisher module in IoT Edge in a ubuntu gateway.
2.  Use OPC Twin webUI to discover, browse OPC UA server.
3.  Using the UI to expand the OPC UA node, then the issue occurred.  Like picture below.

Not showing any OPC tag in Tags node.
![bug123](https://user-images.githubusercontent.com/29178018/65218211-9ba41380-dae8-11e9-8e97-6a6897a7324d.png)

But can see the OPC tag in Tags node while using OPC Client like UAExperts.
![bug456](https://user-images.githubusercontent.com/29178018/65218296-bffff000-dae8-11e9-82e0-5e995e7d49aa.png)

**Expected behavior**
Can successfully expand the OPC node in any OPC UA server just like other OPC client do.

**Screenshots**

Response of expanding the node with  abnormal OPC UA server
![bad_node_Id](https://user-images.githubusercontent.com/29178018/65216768-99d85100-dae4-11e9-8083-c80407f3f28d.png)

Response of expanding the node with  normal OPC UA server
![normal_api_responese](https://user-images.githubusercontent.com/29178018/65216782-a52b7c80-dae4-11e9-8361-6d3120f315a4.JPG)

**Additional context**
The issue behavior is similar with issue #26  , but I think it's different issue. I've used latest industry-service and OPC Twin module to test. and I think issue #26 is resolved now in my testing environment. But the issue which I reported still happened.

**Comment by marcschier** [comment]
Hi @ryan-ms, this looks like a bug resulting from probably double url escaping the namespace uri - once from the GET parameters, and later in the node id parsing.   I will look into it.  

In the meantime could you try the same with the CLI?  Which I believe uses the POST version of browse.

**Comment by ryan-ms** [comment]
Hi @marcschier ,

Thanks for reply.

Sure, I can try the CLI. But actually, I am not very familiar to use CLI, I've referenced the [document](https://github.com/Azure/Industrial-IoT/blob/master/modules/opc-twin/cli/Program.cs) to use CLI.

I can used --list ,  --host command which document mentioned , but not sure what command should I use to browse the OPC node.

Could you please provide me the document or guide me how to use CLI to browse node?
Thank you.

By the way, I've tried use POST method to browse node by using postman, but the result is the same.
![bug789](https://user-images.githubusercontent.com/29178018/65225551-ec6f3880-daf7-11e9-9633-53812e4d772a.png)

**Comment by marcschier** [comment]
Hi @ryan-ms, the issue is that the namespace is not a uri, but just "TagProvider".  I will add a unit test to see that we support this case and whether this is an issue in the GET or sample client.  In the meantime, instead of the above can you use "nsu=TagProvider;s=Tags" as node id using POST in postman?

**Comment by ryan-ms** [comment]
Hi @marcschier ,

Yes, I've tried "nsu=TagProvider;s=Tags" as node id using POST in postman.
And the response is good!
![abc](https://user-images.githubusercontent.com/29178018/66023546-124b0300-e524-11e9-90ec-333ca70b4e72.JPG)

So I assume the Connected Factory or OPC Twin client will update to use POST method in the future, right?!  

Thanks for your response.

**Comment by marcschier** [comment]
Hi @ryan-ms, yes, but I also now have checked in unit tests in master that exercise both POST nad the GET path wtih a bad URI, and they indicated that everything is working ok (I did not need to make any real fixes).  

Connected factory uses the API nuget which is doing POST, but older services, so I assume that this is the issue there and we have fixed something since 2.0.1.   For the webapp, my assumption is that it is doing something wrong such as not url encoding the node id.
